### ISSUE TITLE
An issue with VoiceOver on Safari when downloading tex source

### ISSUE THREAD

**Issue by SBBCornell** [arXiv/html_feedback#654]
Posting this issue on behalf of a reader who uses VoiceOver:

"The browser I used is Safari on a Macbook running on Mac Os Monterey 12. I tried with Firefox instead and this time every thing was ok."

**Issue:**
_"In recent days, I have been unable to download TeX source of papers posted on arXiv (for example on the link below).
Both “TeX Source” and “Other Format>Download Source” links do not work (VoiceOver informs me that there is something that has been downloaded but when I go to the Download Folder there is nothing in it). 

However, I succeeded in downloading the tex source of the arXiv paper https://arxiv.org/abs/2302.07307. So I tried again the arXiv paper mentioned in my previous mails but the problem was still the same: I heard Voiceover telling me something was downloaded but when I went to the Directory folder there was nothing in it. 

In conclusion, downloading source tex with Safari works well for me with some papers but not with others!"_

- Paper with issue: https://arxiv.org/abs/2311.11024
- Paper that downloads normally: https://arxiv.org/abs/2302.07307
